Doppler.HtmlEditorApi/Doppler.HtmlEditorApi.csproj

```
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.90" />
```

Por lo que veo, existe una versión posterior disponible:
```suggestion
    <PackageReference Include="Dapper" Version="2.0.123" />
```
Se utiliza la `2.0.90` por algún motivo en particular?

---

No ningún motivo.

---

Este cambio ya ha sido implementado 💯